mimic/model/identity_errors.py

```diff
@@ -0,0 +1,74 @@
+"""
+Errors for the Identity Models
+"""
+
+
+class EndpointTemplateExceptions(Exception):
```

Comment:
Style nitpick; class names should be singular; except in rare cases where they represent a collection, and this one doesn't.